helm/qontract-reconcile/templates/template.yaml

```diff
@@ -138,6 +149,13 @@ objects:
                 message "\`\`\`[{{ $integration.name }}] %s\`\`\`"
               </store>
               {{- end }}
+              {{- if $logs.googleChat }}
+              <store>
+                @type teams
```

Comment:
I am confused by type teams here.

Reply:
Check the fluentd PR for a more detailed explanation: https://github.com/app-sre/fluentd/pull/4

Perhaps I should include a comment here for visibility?

Reply:
That would be helpful I think.
So googlechat is compatible with Teams API? cause the github page of plugins says, it's for MS Teams: https://github.com/nyamairi/fluent-plugin-teams

Reply:
They should rename this plugin generalized-webhook-format-push-http-plugin. It's  type converting and json packaging log messages to be sent to http webook apis.

The http plugin with fluentd - I spent many days with this trying to get json payloads formatted properly. Did not have much look.

The fluent-plugin-google-chat requires gcp account  access - not available in FedRAMP and also this level of complexity is not needed 
https://github.com/treasuryspring/fluent-plugin-google-chat

This left me to search for other fluentd plugins which could be used in a generalized webhook api fashion - after review the teams one is doing the trick.

Reply:
Thank you for the explanation! I really think that's worth a comment :smiley: